### DISCUSSION TITLE
unable to connect to IMAP

### DISCUSSION THREAD

**Discussion by kingofcold** [uvdesk/community-skeleton#768] [General]
Hi all,

I am using office365 email address with app password and imap is enabled.

getting the below error when trying:
 php bin/console uvdesk:refresh-mailbox support@company.com
 Establishing connection with mailbox

  - Failed to establish connection with mailbox

  Warning: imap_open(): Couldn't open stream {outlook.office365.com:993/imap/ssl}INBOX

  0: Retrying PLAIN authentication after AUTHENTICATE failed.

  1: Retrying PLAIN authentication after AUTHENTICATE failed.

  2: Can not authenticate to IMAP server: AUTHENTICATE failed.

SMTP works fine, and UV desk able to send emails but email to ticket not working.
using latest ver of uvdesk on Ubuntu
PHP 8.1 and extensions are enabled.
imap_server:

                host: '{outlook.office365.com:993/imap/ssl}INBOX'

tried host: outlook.office365.com, and outlook.office365.com:993, and outlook.office365.com:993/ssl

not sure what else to try

**Comment by papnoisanjeev** [comment]
@kingofcold 

We have released a new version (v1.1.7) with Microsoft modern app support as old PLAIN authentication has been removed.
Check [blog](https://www.uvdesk.com/en/how-to-integrate-microsoft-app-to-your-opensource-uvdesk/) for further process.
